my_enumerable.rb

```diff
@@ -0,0 +1,27 @@
+module MyEnumerable
+  def all?
+    result = []
+    @list.each do |element|
+      result.push(element) if yield element
+    end
+    puts result.length == @list.length
+  end
+
+  def any?
+    @list.each do |element|
+      if yield element
+        puts true
+        return true
+      end
+    end
+    puts false
+  end
+
+  def filter
+    result = []
+    @list.each do |element|
+      result.push(element) if yield element
+    end
+    print result
```

Comment:
- [ ] As requested by the previous, Please remove the `puts` and `prints` command inside the functions and just return the value. In the commit history, I can see you fixed the issues and then reverted back. Please make sure you implement the requested changes by the previous reviewer.